spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/DataFlowServerUtil.java

```diff
@@ -27,7 +27,9 @@
  * @author Mark Fisher
  * @author Ilayaperumal Gopinathan
  * @author Gunnar Hillert
+ * @deprecated as of 1.7.  ApplicationType determined at parse time.
  */
+@Deprecated
```

Comment:
We can probably deprecate the `CannotDetermineApplicationTypeException` class as well.